src/sds_data_model/vector.py

```diff
@@ -133,11 +133,16 @@ def get_col_dtype(
         self: _VectorTile,
         column: str,
     ) -> str:
```

Comment:
We're now returning a `str` or `None`, so the return type should be `Optional[str]`.

```suggestion
    ) -> Optional[str]:
```